astropy/io/votable/tree.py

```diff
@@ -1115,22 +1115,25 @@ def to_table_column(self, column):
         column.meta['values'] = meta
 
     def from_table_column(self, column):
-        if 'values' not in column.meta:
-            return
-
-        meta = column.meta['values']
-        for key in ['ID', 'null']:
-            val = meta.get(key, None)
-            if val is not None:
-                setattr(self, key, val)
-        if 'min' in meta:
-            self.min = meta['min']['value']
-            self.min_inclusive = meta['min']['inclusive']
-        if 'max' in meta:
-            self.max = meta['max']['value']
-            self.max_inclusive = meta['max']['inclusive']
-        if 'options' in meta:
-            self._options = list(meta['options'].items())
+        if isinstance(column, Quantity):
```

Comment:
This is a bit confusing -- it makes the object type of `column` ambiguous, where it might be a `Column` or it might be an `Info` object.  In this case those two happen to have the same `meta` attribute, but you want to avoid tricks like this.

Fortunately a true `Column` object has the same `info` attribute as any mixin, so the easy way out is just changing `column.meta` to `column.info.meta` everywhere.   Indeed that idiom is always required for writing "mixin-safe" code.